File: 0.C_CPP_JAVA Programs/JAVA/Program15.java
/*5. Write java program which accept N numbers from user and return product of all odd elements.
Input :  N :   6     Elements : 15 66 3 70 10 88
Output : 45
Input :  N :   6     Elements : 44 66 72 70 10 88
Output : 0

 */

import java.lang.*;
import java.util.*;
class DemoArray
{
    int Check(int arr[])
    {
        int iCount=0;
        int iMult=1;
        for(int i=0;i<arr.length;i++)
        {
           if(arr[i]%2!=0)
           {
               iMult=iMult*arr[i];
           }
        }
        return iMult;
    }
}
class Demo
{
    public static void main(String arg[])
    {

         Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the size");
        int size=sobj.nextInt();

        System.out.println("Enter element");

        int arr[]=new int[size];

        for (int i=0;i<arr.length;i++)
        {
            arr[i]=sobj.nextInt();
        }

        DemoArray dobj=new DemoArray();

        int ret=dobj.Check(arr);
        System.out.println("Your Product is " + ret);
    }
}

File: 1.Python/Program15.py
#3. Write a program which accept one number from user and return its factorial.
# Input :  5
# Output : 120
def Fact(No):
     iCnt=1
     i=1
     for iCnt in range(1,No+1,1):
        i=i*iCnt
     return i

def Main():
   No=0
   iRet=0

   print("Enter the one number:")
   No =int(input())

   iRet=Fact(No)
   print("Your factorial is: ", iRet)

if __name__ == "__main__":
   Main()